### DISCUSSION TITLE
vcpkg search failed!!!

### DISCUSSION THREAD

**Discussion by whynotAC** [microsoft/vcpkg#42180] [Q&A] [answered]
> .\vcpkg.exe search opencv
> E:\github\vcpkg\.git: error: .git port manifest missing (no vcpkg.json or CONTROL file)
> E:\github\vcpkg\.github: error: .github port manifest missing (no vcpkg.json or CONTROL file)
> E:\github\vcpkg\buildtrees: error: buildtrees port manifest missing (no vcpkg.json or CONTROL file)
> E:\github\vcpkg\docs: error: docs port manifest missing (no vcpkg.json or CONTROL file)
> E:\github\vcpkg\downloads: error: downloads port manifest missing (no vcpkg.json or CONTROL file)
> E:\github\vcpkg\installed: error: installed port manifest missing (no vcpkg.json or CONTROL file)
> E:\github\vcpkg\packages: error: packages port manifest missing (no vcpkg.json or CONTROL file)
> E:\github\vcpkg\ports: error: ports port manifest missing (no vcpkg.json or CONTROL file)
> E:\github\vcpkg\scripts: error: scripts port manifest missing (no vcpkg.json or CONTROL file)
> E:\github\vcpkg\toolsrc: error: toolsrc port manifest missing (no vcpkg.json or CONTROL file)
> E:\github\vcpkg\triplets: error: triplets port manifest missing (no vcpkg.json or CONTROL file)
> E:\github\vcpkg\versions: error: versions port manifest missing (no vcpkg.json or CONTROL file)
> note: updating vcpkg by rerunning bootstrap-vcpkg may resolve this failure.

I can't figure out what's the problem as I modified nothing.

**Comment by autoantwort** [comment, marked as answer]
Can't repo. The following works:
1. `git clone git@github.com:microsoft/vcpkg.git`
2. `cd vcpkg`
3. `bootstrap-vcpkg.bat`
4. `vcpkg.exe search opencv`

**Comment by fjcheng** [comment]
I had the same problem

```shell
D:\dev\vcpkg>vcpkg search vtk
D:\dev\vcpkg\ports\strawberry-perl: error: strawberry-perl port manifest missing (no vcpkg.json or CONTROL file)
```
After deleting directory strawberry-perl, it can run. Why?

**Reply by autoantwort** [reply]
Because in the `strawberry-perl` directory was no manifest file
